c/driver/postgresql/connection.cc

```diff
@@ -654,10 +652,23 @@ AdbcStatusCode PostgresConnectionGetInfoImpl(const uint32_t* info_codes,
         RAISE_ADBC(
             AdbcConnectionGetInfoAppendString(array, info_codes[i], "PostgreSQL", error));
         break;
-      case ADBC_INFO_VENDOR_VERSION:
-        RAISE_ADBC(AdbcConnectionGetInfoAppendString(
-            array, info_codes[i], std::to_string(PQlibVersion()).c_str(), error));
```

Comment:
For future reference it appears PQlibVersion returns the libpq version of the client, not the server

Reply:
Ah...that makes sense, yeah.